### PR TITLE
Add CAVLC B-Frame decoder support without build errors. 

### PR DESCRIPTION
**Pull request by xiaotiansf**
This is basically a copy of #PR 3058

### CODE REVIEW

File: codec/decoder/core/src/decode_slice.cpp [hunk 1]
```diff
@@ -52,6 +52,11 @@
 
 namespace WelsDec {
 
+#if defined(_DEBUG)
+FILE* pFile = fopen ("Y:/p4/CodeProjects/Upwork/screencast/xiaotiansf/openh264-1/MV_CAVLC.txt", "w");
```
Comment (mstorsjo) [inline codec/decoder/core/src/decode_slice.cpp]: There's no point in first adding this in one commit and then disabling it in a later commit.

Comment (xiaotianshi2) [reply codec/decoder/core/src/decode_slice.cpp]: @thaytan Yes, I'll appreciate it if you can help me  with git operations to generate 2 clean branches for PRs.

Comment (sijchen) [reply codec/decoder/core/src/decode_slice.cpp]: @xiaotianshi2  you can consider doing squash to merge commits. https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

Comment (sijchen) [reply codec/decoder/core/src/decode_slice.cpp]: @xiaotianshi2 and please separate commits for different purpose to different PRs, you can do that by starting from a clean branch, then "cherry-pick" the needed commits.

**Comment by sijchen** [comment]
close this for clearer commit requirement.

**Pull request closed without merging**